esmvaltool/recipes/recipe_impact.yml

```
@@ -0,0 +1,125 @@
+  - {mip: Amon, project: CMIP6, exp: [historical, ssp585], dataset: AWI-CM-1-1-MR, ensemble: r1i1p1f1, grid: gn, tag: model}
+  - {mip: Amon, project: CMIP6, exp: [historical, ssp585], dataset: BCC-CSM2-MR, ensemble: r1i1p1f1, grid: gn, tag: model}
+  - {mip: Amon, project: CMIP6, exp: [historical, ssp585], dataset: CAMS-CSM1-0, ensemble: r1i1p1f1, grid: gn, tag: model}
+  - {mip: Amon, project: CMIP6, exp: [historical, ssp585], dataset: CanESM5, ensemble: r1i1p1f1, grid: gn, tag: model}
+  - {mip: Amon, project: CMIP6, exp: [historical, ssp585], dataset: CESM2-WACCM, ensemble: r1i1p1f1, grid: gn, tag: model}
+  # - {mip: Amon, project: CMIP6, exp: [historical, ssp585], dataset: CESM2, ensemble: r1i1p1f1, grid: gn, tag: model}
+  - {mip: Amon, project: CMIP6, exp: [historical, ssp585], dataset: FGOALS-g3, ensemble: r1i1p1f1, grid: gn, tag: model}
+  - {mip: Amon, project: CMIP6, exp: [historical, ssp585], dataset: FIO-ESM-2-0, ensemble: r1i1p1f1, grid: gn, tag: model}
+  - {mip: Amon, project: CMIP6, exp: [historical, ssp585], dataset: MIROC6, ensemble: r1i1p1f1, grid: gn, tag: model}
+  - {mip: Amon, project: CMIP6, exp: [historical, ssp585], dataset: MPI-ESM1-2-HR, ensemble: r1i1p1f1, grid: gn, tag: model}
+  - {mip: Amon, project: CMIP6, exp: [historical, ssp585], dataset: MPI-ESM1-2-LR, ensemble: r1i1p1f1, grid: gn, tag: model}
+  - {mip: Amon, project: CMIP6, exp: [historical, ssp585], dataset: MRI-ESM2-0, ensemble: r1i1p1f1, grid: gn, tag: model}
+  - {mip: Amon, project: CMIP6, exp: [historical, ssp585], dataset: NESM3, ensemble: r1i1p1f1, grid: gn, tag: model}
+  - {mip: Amon, project: CMIP6, exp: [historical, ssp585], dataset: NorESM2-LM, ensemble: r1i1p1f1, grid: gn, tag: model}
+  - {mip: Amon, project: CMIP6, exp: [historical, ssp585], dataset: NorESM2-MM, ensemble: r1i1p1f1, grid: gn, tag: model}
+
+
+observations: &observations
+  - {mip: Amon, dataset: ERA5, project: native6, type: reanaly, version: '1', tier: 3, tag: observations}
+
+preprocessors:
+
+  calculate_anomalies:
+    custom_order: true
+    extract_region: &region
+      start_longitude: 0.0
+      end_longitude: 39.0
+      start_latitude: 30.0
+      end_latitude: 76.25
+    area_statistics:
+      operator: mean
+    anomalies:
+      period: full
+      reference: &reference_period
+        start_year: 1981
+        start_month: 1
+        start_day: 1
+        end_year: 2010
+        end_month: 12
+        end_day: 31
+      standardize: false
+    extract_time:  # Future period
+      start_year: 2035
+      start_month: 1
+      start_day: 1
+      end_year: 2065
+      end_month: 12
+      end_day: 31
+    climate_statistics:
+      operator: mean
+
+  prepare_for_bias_calculation:
+    regrid:
+      scheme: linear
+      target_grid: 2.5x2.5
+        # <<: *region
+        # lon_step: 2.5
+        # lat_step: 2.5
+    extract_region:
+      <<: *region
+    extract_time:
+      <<: *reference_period
+
+diagnostics:
+  quick_insights:
```

Do you think this can be changed to something more descriptive, like:

```suggestion
  model_bias:
```

---

It's not just bias. "calculate bias and visualize bias and change" is a bit too long. Maybe I can change the description to "provide quick insights into the behaviour of each model by plotting the bias with respect to observations and change with respect to the reference period." Like that, the "quick insights" is immediately explained.

---

Maybe `plot_bias_and_change` or `calculate_bias_and_change`? `quick_insights` does not say anything about what is calculated or plotted.

---

made it `bias_and_change` 👍